js/script.js

```diff
@@ -0,0 +1,98 @@
+const formInputs = document.querySelectorAll(".form__input-box");
+const formEmail = document.querySelector("#form__email");
+const formPassword = document.querySelector("#form__password");
+const formSubmit = document.querySelector(".form__submit");
+
+// error 종류에 따라 메시지 추가
+const errorMsgs = {
+  emptyInput: {
+    email: "이메일을 입력해주세요.",
+    password: "비밀번호를 입력해주세요.",
+    text: "비밀번호를 입력해주세요.",
+  },
+  invalidInput: {
+    email: "올바른 이메일 주소가 아닙니다.",
+  },
+  invalidLogin: {
+    email: "이메일을 확인해주세요.",
+    password: "비밀번호를 확인해주세요.",
+  },
+};
+
+function printErrorMsg(error, type, target) {
+  removeErrorMsg(target);
+  const errorMsg = document.createElement("p");
+  errorMsg.textContent = errorMsgs[error][type];
+  errorMsg.classList.add("error-msg");
+  target.after(errorMsg);
+}
+
+function removeErrorMsg(target) {
+  if (
+    target.nextElementSibling &&
+    target.nextElementSibling.classList.contains("error-msg")
+  ) {
+    target.nextElementSibling.remove();
+  }
+}
+
+// input에 값이 존재하는지 확인
+function checkEmptyInput(event) {
+  if (!event.target.value) {
+    event.target.classList.add("form__input-box__error");
+    printErrorMsg("emptyInput", event.target.type, event.target);
+  } else {
+    event.target.classList.remove("form__input-box__error");
+    removeErrorMsg(event.target);
+  }
+}
+
+// input 양식에 맞게 값을 입력했는지 확인
+function validateInput(event) {
+  if (event.target.type == "email" && event.target.value) {
+    const emailTest =
+      /^([\w-]+(?:\.[\w-]+)*)@((?:[\w-]+\.)*\w[\w-]{0,66})\.([a-z]{2,6}(?:\.[a-z]{2})?)$/i;
+    if (!emailTest.test(event.target.value)) {
+      printErrorMsg("invalidInput", event.target.type, event.target);
+    }
+  }
+}
+
+// 로그인 시도 확인
+const account = {
+  "test@codeit.com": "codeit101",
+};
```

Comment:
account 객체가 email, password 로 이루어진 구조라는 것은 이해하였는데 꼭 email 이 key 가 되지 않아도 괜찮아요.

```suggestion
const account = {
  email: "test@codeit.com",
  password: "codeit101",
};
```

email, password 로 분리해서 정보를 보다 쉽게 접근할 수 있게 하면 좋을 것 같습니다. (+ 확장성)

Reply:
이후 다른 계정들을 추가할 수 있도록 다음과 같이 변경하는 것은 어떻게 생각하시는지 궁금합니다...! 현재는 테스트 계정 하나만 필요한 상황이기에 따로 묶어둘 필요는 없는 걸까요?
```
const account = {
  test: {
    email: "test@codeit.com",
    password: "codeit101",
  }
};
```